s/api/express-middleware.ts

```diff
@@ -43,6 +43,12 @@ export async function expressMiddleware({
 	const execute = renraku.servelet(api)
 
 	const middleware: express.RequestHandler = async(req, res, next) => {
+		if (req.url.endsWith("/health")) {
+			console.log(`⚕️ health check`)
+			res.setHeader("Content-Type", "text/plain; charset=utf-8")
+			res.statusCode = 200
+			res.end(Date.now().toString())
+		}
 		const userIntegration = await authUser(req)
```

Comment:
oh, you need to wrap the rest of the code in the middleware in an `else` block

Reply:
okay. I will implement that now.

Reply:
Done